modules/machines-edge-machine-pool.adoc

```diff
@@ -0,0 +1,103 @@
+
+:_content-type: CONCEPT
+[id="machines-edge-machine-pool_{context}"]
+= The edge machine pool
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. However, due to the image type and size limitations on Local Zone resources, the default instance type that is created can vary from the traditional worker pool.
+
+The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools. Depending on the availability of the location, the system chooses one of those instances. The preferred list of instance types for the edge compute pool includes:
+
+* `m6i.xlarge`
+* `m5.xlarge`
+* `c5d.2xlarge`
```

Comment:
I think we can stop on `Depending on the availability of the location`.

This list, order, or logic can be changed over time on the installer, let's avoid maintenance on the docs. 


```suggestion
The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools, depending on the availability of the location.
```

cc @yunjiang29 (wdyt?)

Reply:
@mtulio What is the difference between this section and https://github.com/openshift/openshift-docs/pull/57057, I think maintaining one list is enough.